framework/packages/standards/dex/Cargo.toml

```diff
@@ -17,8 +17,9 @@ crate-type = ["cdylib", "rlib"]
 [features]
 default = ["export"]
 export = []
-testing = ["dep:cw-orch"]
+testing = ["dep:cw-orch", "cw-orch/daemon"]
```

Comment:
Why do we need daemon here?

Reply:
It's used in `standards/dex/src/tests.rs`

Reply:
https://github.com/AbstractSDK/abstract/blob/14b695bf19e1e9082f9dd8ff32e82fbd72d82a46/framework/packages/standards/dex/src/tests.rs#L9-L10